endpoints/openrtb2/auction.go

```diff
@@ -2319,9 +2319,11 @@ func (deps *endpointDeps) processStoredRequests(requestJson []byte, impInfo []Im
 
 // parseImpInfo parses the request JSON and returns impression and unmarshalled imp.ext.prebid
 func parseImpInfo(requestJson []byte) (impData []ImpExtPrebidData, errs []error) {
+	// ignore the error such as '&errors.errorString{s:"Unknown value type"}'
```

Comment:
Should this comment be here? Since the line directly beneath does handle the error.

Reply:
> Should this comment be here? Since the line directly beneath does handle the error.

But does it actually? The line underneath is:
`if impArray, dataType, _, err := jsonparser.Get(requestJson, "imp"); err == nil && dataType == jsonparser.Array {`
it doesn't handle cases where `err != nil` OR `dataType != jsonparser.Array`, right?  It just returns at the end of the function (L2337).  Please let me know if I'm missing something obvious 😄

Reply:
Ah yes you're right, then do you mind me asking why did you add this comment? For my own understanding